doc/source/install.rst

```diff
@@ -0,0 +1,8 @@
+Installation
+===============
+
+Tenks has dependencies on Ansible roles that are hosted by Ansible Galaxy.
+Given that your virtualenv of choice is active and Ansible (>=2.6) is
```

Comment:
Ansible is in requirements.txt.

Reply:
True. What is the preferred procedure for installation then? If we install the package from PyPI with pip, I suppose we still need to clone the Tenks repo for the playbooks that we're referencing in the ansible-playbook commands?

Reply:
Yes. For kayobe we say to clone the repo, create a virtualenv, then `pip install .`, which pulls in requirements.txt.

Reply:
Sounds reasonable, I'll say that here as well.